README.md

```diff
@@ -48,3 +64,6 @@ poetry run python/__main__.py [-h] [--schema SCHEMA] [--inputdir INPUTDIR] [--ou
 ## Maybe later / open for discussion
 
 * A source file contains questions and must be in the sources folder and contain a reference to a known schema.
+
+* The project contains own (custom) modules. To make that work, for now the source folder has to be added to PYTHONPATH
+  f.e. export PYTHONPATH=/path/to/sourcefolder
```

Comment:
I'd leave this out

Reply:
Agree, as later tad will be a package by itself, just like that you would run `tad` directly instead of `poetry run python -m`. For now this can still be the way of working though internally before it is a package.